code/modules/power/supermatter/supermatter.dm

```diff
@@ -355,6 +355,8 @@
 /obj/machinery/power/supermatter/proc/Consume(atom/movable/AM)
 	if(istype(AM, /mob/living))
 		var/mob/living/user = AM
+		if(user.ckey)
+			log_attack("[src] has consumed [pulledby ? "(being pulled by [pulledby])" : "last touched by [fingerprintslast]"] [AM]/([key_name(user)])")
```

Comment:
Do you mean if the var doesn't exist?
Shards have fingerprints otherwise there would be a compile error